internal/registry/types.iterations.go

```diff
@@ -65,10 +65,12 @@ func GetGitFingerprint(opts IterationOptions) (string, error) {
 	})
 
 	if err != nil {
-		return "", fmt.Errorf("Packer was unable to load a git sha. "+
-			"If your Packer template is not in a git repo, please add a unique "+
-			"template fingerprint using the env var HCP_PACKER_BUILD_FINGERPRINT. "+
-			"Error: %s", err)
+		return "", fmt.Errorf("Unique fingerprint required for iteration\nPacker " +
+			"cannot find a git SHA to use as a unique fingerprint for this " +
+			"HCP Packer build. If your Packer template is not within a git " +
+			"managed directory, you can set the HCP_PACKER_BUILD_FINGERPRINT " +
+			"environment variable. The fingerprint must be less than 32 " +
+			"characters and can contain letters and numbers.")
```

Comment:
The following is the error message that gets displayed to the user, which is a bit repetitive. We can clean this up a bit. 

```
Error: Iteration initialization failed

Initialization of the iteration failed with the following error message: Unique
fingerprint required for iteration
Packer cannot find a git SHA to use as a unique fingerprint for this HCP Packer
build. If your Packer template is not within a git managed directory, you can
set the HCP_PACKER_BUILD_FINGERPRINT environment variable. The fingerprint must
be less than 32 characters and can contain letters and numbers..
Contact HCP Packer support for further assistance
```